lib/optimizely.rb

```diff
@@ -269,6 +276,10 @@ def is_feature_enabled(feature_flag_key, user_id, attributes = nil)
         end
 
         @logger.log(Logger::INFO, "Feature '#{feature_flag_key}' is enabled for user '#{user_id}'.")
+        @notification_center.fire_notifications(
+            NotificationCenter::NOTIFICATION_TYPES[:FEATURE_ACCESSED],
```

Comment:
This needs to be changed. 
1. Fire NOTIFICATION_TYPES[:FEATURE_EXPERIMENT] when the source is DECISION_SOURCE_EXPERIMENT
2. Fire NOTIFICATION_TYPES[:FEATURE_ROLLOUT] when the source is DECISION_SOURCE_ROLLOUT

Reply:
Done